configure

```diff
@@ -937,22 +959,41 @@ def configure_node(o):
   else:
     o['variables']['node_enable_v8_vtunejit'] = 'false'
 
+  if flavor != 'linux' and (options.enable_pgo_generate or options.enable_pgo_use):
+    raise Exception(
+      'The pgo option is supported only on linux.')
+
+  if flavor == 'linux':
+    if options.enable_pgo_generate or options.enable_pgo_use:
+      version_checked = (5, 4, 1)
+      if check_gcc_version(version_checked) == False:
+        version_checked_str = ".".join(map(str, version_checked))
+        raise Exception(
+          'The options --enable-pgo-generate and --enable-pgo-use options '
```

Comment:
Duplicate 'options', should probably be `'The options --enable-pgo-generate and --enable-pgo-use '`?

Reply:
Agree and tried to resolve it.